CK3toEU4/Source/Mappers/RegionMapper/RegionMapper.h

```diff
@@ -0,0 +1,48 @@
+#ifndef EU4_REGIONS_H
+#define EU4_REGIONS_H
+
+#include "Area.h"
+#include "Parser.h"
+#include "Region.h"
+#include "SuperRegion.h"
+#include <map>
+
+class Configuration;
+namespace mappers
+{
+class RegionMapper: commonItems::parser
+{
+  public:
+	RegionMapper() = default;
+	virtual ~RegionMapper() = default;
```

Comment:
Why the virtual destructor. Are you inheriting from this class (including mocks)? If not, delete the virtual.

At which point you can just delete all the default big six.